data_collection/web/advanced.rst

```diff
@@ -24,3 +71,81 @@ The following global names are used by PPAS JavaScript tracker. Websites that wi
 * ``piwik_log``
 * ``piwik_track``
 * ``sevenTag``
+
+
+.. _data-collection-tracker-alternative-configuration:
+
+Alternative multi-domain configurations
+---------------------------------------
+
+Tracking domains and all subdomains
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To track all data between domain and all its subdomains, we must use cookies configured with the following snippet::
+
+    _paq.push(["setTrackerUrl", u+"ppms.php"]);
+    _paq.push(["setSiteId", "XXX-XXX-XXX-XXX-XXX"]);
+
+    // Share the tracking cookie across example.com, www.example.com, subdomain.example.com, ...
+    _paq.push(["setCookieDomain", "*.example.com"]);
+
+    // Tell Piwik the website domain so that clicks on these domains are not tracked as "Outlinks"
+    _paq.push(["setDomains", "*.example.com"]);
+
+    _paq.push(["trackPageView"]);
+
+Tracking multiple domains as one site
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To set up tracking between multiple domains, you must use multiple functions ``setDomains`` to set a list of domains and
+``enableCrossDomainLinking`` to enable cross domain linking::
+
+    _paq.push(["setDomains", domains]);
+
+.. describe:: domains
+
+    **Required** ``array`` Domains array, with wildcards
+
+::
+
+    _paq.push(["enableCrossDomainLinking"]);
+
+Tracking subdirectories of domain as separate websites
+------------------------------------------------------
+
+To differentiate parts of a website as another site, you must configure tracker this way::
+
+    _paq.push(["setSiteId", "App1"]);
+    _paq.push(["setTrackerUrl", u+"ppms.php"]);
+    _paq.push(["trackPageView"]);
+
+Afterwards, you can change configuration for selected paths and track them as another site::
+
+    _paq.push(["setSiteId", "App2"]);
+
+    _paq.push(["setCookiePath", "/data/something_useful"]);
+
+    _paq.push(["setDomains", "example.com/data/something_useful"]);
+
+    _paq.push(["setTrackerUrl", u+"ppms.php"]);
+    _paq.push(["trackPageView"]);
+
+This way, all actions tracked on ``/data/something_useful`` will be tracked for ``App2`` instead of ``App1``.
+
+If you wish to track a group of pages as separate site, you can use the wildcard in the ``setDomains`` function.
+
+Navigation timing page performance metrics
+------------------------------------------
+
+To set up page performance metrics gathering use the
+``setTimingDataSamplingOnPageLoad`` function::
+
+    _paq.push(["setTimingDataSamplingOnPageLoad", updateTimingDataOnPageLoadSampling]);
+
+.. describe:: updateTimingDataOnPageLoadSampling
+
+    **Required** ``integer`` Value between 1 and 100 describing the percentage for data sampling
+
+::
```

Comment:
Same here, we should give an example of usage and give a link to the function in API doc.